roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -173,6 +173,12 @@ spec:
             # Wait for the datastore.
             - name: WAIT_FOR_DATASTORE
               value: "true"
+{% endif %}
+{% if calico_network_backend is defined and calico_network_backend == 'vxlan' %}
+            - name: FELIX_VXLANNVI
+              value: "{{ calico_vxlan_vni | default(4096) }}"
```

Comment:
please put the default value in defaults/main.yml and don't hide it here

Reply:
I've removed the hidden defaults and added them to defauls/main.yml